src/Network/Receive.pm

```diff
@@ -8889,11 +8889,17 @@ sub skill_msg {
 sub msg_string {
 	my ($self, $args) = @_;
 
-	if ($msgTable[++$args->{index}]) { # show message from msgstringtable.txt
-		message "$msgTable[$args->{index}]. Value: $args->{paral}\n", "info";
+	my $index = ++$args->{index};
+	if ($msgTable[$index]) { # show message from msgstringtable.txt
+		message "$msgTable[$index]. Value: $args->{paral}\n", "info";
```

Comment:
i've noticed that before, it had the `++` only in one place. Both in warning and message it was displayed the index without the `++`. But now every part has the plus sign. Can you confirm if the message remains as intended?

Reply:
at this point $index already have the correct value